### DISCUSSION TITLE
Installation help - Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists.

### DISCUSSION THREAD

**Discussion by adriankchan77** [Azure/azure-workload-identity#1079] [General]
Hello! We're trying to install this on our cluster, following these instructions:
https://azure.github.io/azure-workload-identity/docs/installation/mutating-admission-webhook.html

So this command:
helm install workload-identity-webhook azure-workload-identity/workload-identity-webhook 
--version 1.1.0 --namespace azure-workload-identity-system --create-namespace --set azureTenantID=<tenantid>

But getting this error:
Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: ClusterRole "azure-wi-webhook-manager-role" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "workload-identity-webhook"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "azure-workload-identity-system"

What's the problem? Does anyone have any ideas how we can get past this?

Thanks!

**Comment by sozercan** [comment]
@adriankchan77 sounds like you already have `azure-wi-webhook-manager-role` cluster role in your cluster. Can you try removing it and redeploying?

**Reply by adriankchan77** [reply]
Thanks. Looking into it, it turns out that when we create the cluster with --enable-workload-identity, an integrated version is being used. And we don't need to install it from the Helm chart.

https://github.com/Azure/azure-workload-identity/issues/856

**Reply by sozercan** [reply]
That's correct. If you create an AKS cluster with `--enable-workload-identity`, AKS Workload Identity addon will be deployed and you don't need to install the OSS component seperately. I'll close this as resolved.
